freqtrade/exchange/exchange.py

```diff
@@ -732,10 +734,11 @@ def amount_to_precision(self, pair: str, amount: float) -> float:
 
     def price_to_precision(self, pair: str, price: float) -> float:
         """
-        Returns the price rounded up to the precision the Exchange accepts.
-        Rounds up
+        Returns the price rounded to the precision the Exchange accepts.
+        The default price_rounding_mode in conf is ROUND_UP
         """
-        return price_to_precision(price, self.get_precision_price(pair), self.precisionMode)
+        return price_to_precision(price, self.get_precision_price(pair),
+                                  self.precisionMode, self._price_rounding_mode)
```

Comment:
this won't work reliably throughout the bot.

`exchange.price_to_precision()` is only used from within the exchange class (it's a local helper, just as the other methods around here).

neither from trade-model, nor from backtesting we'll call this (or from trade_model, we won't be able to call this, either) - but we call the actual method instead.

As is - it's therefore incomplete - and will only apply to to probably less than half of the actually used locations.